lutris/runners/wine.py

```diff
@@ -725,6 +739,7 @@ def prelaunch(self):
         self.sandbox(prefix_manager)
         self.set_regedit_keys()
         self.setup_x360ce(self.runner_config.get("x360ce-path"))
+        self.setup_nine(self.runner_config.get("gallium_nine"))
```

Comment:
put this in a try / except with an exception handler please

Reply:
Exception handler added